impexp-core/src/main/java/org/citydb/core/operation/exporter/database/content/DBTunnelHollowSpace.java

```diff
@@ -119,61 +122,76 @@ public DBTunnelHollowSpace(Connection connection, CityGMLExportManager exporter)
 				}
 				surfaceADEHookTables = addJoinsToADEHookTables(TableEnum.THEMATIC_SURFACE, thematicSurface);
 			}
-			if (projectionFilter.containsProperty("interiorFurniture", tunnelModule)) {
-				CombinedProjectionFilter tunnelFurnitureProjectionFilter = exporter.getCombinedProjectionFilter(TableEnum.TUNNEL_FURNITURE.getName());
+			if (projectionFilter.containsProperty("hollowSpaceInstallation", tunnelModule)) {
```

Comment:
Here should be "interiorFurniture" instead of "hollowSpaceInstallation"

Reply:
fixed wirh 5539d4066680714a24f854506a010ba7dfdc878d